### PR TITLE
Sends learner name to lms

### PR DESCRIPTION
**Pull request by garyplard**
# Problem

We need to retrieve the learner name in addition of his id

# Solution

Send it through query params to the lms

### CODE REVIEW

**Comment by garyplard** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #41** <a href="https://app.graphite.dev/github/pr/Didask/scol-r/41?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Didask/scol-r/41?utm_source=stack-comment).

File: src/loadContent.js [hunk 1]
```diff
@@ -112,19 +112,19 @@ function loadContent() {
...
-    "&lms_origin=" +
-    encodeURIComponent(location.origin) +
-    "&data_from_lms=" +
-    ADAPTER.getDataFromLMS();
+    (sourceUrlParser.search.startsWith("?") ? "&" : "?") +
+    "scorm" +
+    `&learner_id=${learnerId}` +
+    `&learner_name=${learnerName}` +
+    `&lms_origin=${encodeURIComponent(location.origin)}` +
+    `&data_from_lms=${ADAPTER.getDataFromLMS()}`;
```
Comment (Alounv) [inline src/loadContent.js:127]: Est-ce que ça peut se tester avec un test unitaire ?

**Review by Alounv — APPROVED** [review]

File: src/loadContent.js (hunk 1 above)
Comment (garyplard) [reply src/loadContent.js:127]: Bonne idée, je les ai ajoutés sur la PR suivante (je me rends compte que j'aurais pu le faire ici, mais maintenant que c'est fait 🤷🏻‍♂️)

**Comment by garyplard** [comment]
### Merge activity

* **Feb 14, 6:05 AM EST**: @garyplard started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Didask/scol-r/41).

**Pull request merged**